docs/source/index.rst

```diff
@@ -32,6 +32,7 @@ architectures, and common image transformations for computer vision.
    :caption: Package Reference
 
    datasets
+   feature_extraction
```

Comment:
It looks like `feature_extraction` is a submodule or a `folder` in torchvision, but I think the files are placed under `models/feature_extraction`

Reply:
Wasn't keen on putting it in the models docs as that seems to have a consistent theme of talking about models, how to load them, and their metrics. And I'm a noob with auto-docs so didn't know if I could nest it in they table of contents but still have its own page. Any suggestions?

Reply:
Re. your comment below:

> Probably there is small confusion in placing the files?

Yes probably :)

Reply:
I'm pretty much noob too :smile: Probably the maintainers here would tell you better solution for docs.